### PR TITLE
Update Vim talk

### PR DESCRIPTION
**Pull request by Advill**
Just a few quick updates for this year. Mostly spelling fixes, but at least two new sections and one link fix.

### CODE REVIEW

File: Vim.md [hunk 1]
```diff
@@ -12,39 +12,39 @@ require the user to remove their hands from the home row.
 
 ### Getting started with Vim
 
-To start vim without opening a specific file simply type `vim` on the terminal.
-To open a file, write `vim filename`, in the case of this file, one would type
-`vim vim.md`, like you likely already have.
+To start Vim without opening a specific file simply type `Vim` on the terminal.
+To open a file, write `Vim filename`, in the case of this file, one would type
+`Vim Vim.md`, like you likely already have.
```
Comment (ripleymj) [inline Vim.md]: I'm not sure vim should be capitalized when referring to launching it from the terminal.

**Comment by ripleymj** [comment]
Do you want to merge this before the meeting so people see the latest, or hold it until after in case you notice something while presenting it?

File: Vim.md (hunk 1 above)
Comment (Advill) [reply Vim.md]: whoops, nice catch

**Comment by Advill** [comment]
I think just merge now, makes it easier to see the markdown stuff. Unless you or someone else has strong opinions to the contrary

File: Vim.md (hunk 1 above)
Comment (ripleymj) [reply Vim.md]: I'm thinking line 15 and 17 too

**Comment by ripleymj** [comment]
LGTM, but I'll leave it open for additional comments until I head over there.

**Comment by Advill** [comment]
Sounds good.

**Pull request merged**